# HG changeset patch
# User Jacob Dawid <jacob.dawid@googlemail.com>
# Date 1327967274 -3600
#      Tue Jan 31 00:47:54 2012 +0100
# Branch gui
# Node ID faece6b2ab90bf637244001a6fe0a8a634cd3075
# Parent  fdd5828e2214a24eb67479061e77c1a41f7ac195
Corrected license headers in all files to GPL.

* README: Deleted because it was outdated anyways.

diff --git a/gui/README b/gui/README
deleted file mode 100644
--- a/gui/README
+++ /dev/null
@@ -1,24 +0,0 @@
-/* OctaveGUI - A graphical user interface for Octave
- * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License, or (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program. If not, see <http://www.gnu.org/licenses/>.
- */
-
-Octave GUI is a graphical user interface for Octave with readline support.
-
-If you have any problems, suggestions or ideas, feel free to drop me a mail at
-jacob.dawid@googlemail.com or visit the official channel #octave on FreeNode.
-
-- Jacob Dawid
-
diff --git a/gui/get-qirc.sh b/gui/get-qirc.sh
--- a/gui/get-qirc.sh
+++ b/gui/get-qirc.sh
@@ -1,24 +1,24 @@
 #!/bin/bash
 
 # OctaveGUI - A graphical user interface for Octave
 # Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
+# it under the terms of the GNU General Public License as
 # published by the Free Software Foundation, either version 3 of the
 # License, or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
+# GNU General Public License for more details.
 #
-# You should have received a copy of the GNU Affero General Public License
+# You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
 qirc_dir="qirc";
 echo "Updating qirc..";
 
 # Check whether we have already cloned the repository:
 if [ -d $qirc_dir ]; then
diff --git a/gui/get-qterminal.sh b/gui/get-qterminal.sh
--- a/gui/get-qterminal.sh
+++ b/gui/get-qterminal.sh
@@ -1,24 +1,24 @@
 #!/bin/bash
 
 # OctaveGUI - A graphical user interface for Octave
 # Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
+# it under the terms of the GNU General Public License as
 # published by the Free Software Foundation, either version 3 of the
 # License, or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
+# GNU General Public License for more details.
 #
-# You should have received a copy of the GNU Affero General Public License
+# You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
 qterminal_dir="qterminal";
 echo "Updating qterminal..";
 
 # Check whether we have already cloned the repository:
 if [ -d $qterminal_dir ]; then
diff --git a/gui/msvc-debug.pri b/gui/msvc-debug.pri
--- a/gui/msvc-debug.pri
+++ b/gui/msvc-debug.pri
@@ -1,8 +1,24 @@
+# OctaveGUI - A graphical user interface for Octave
+# Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
 win32-msvc* {
 	CONFIG += release warn_off
 	QMAKE_CXXFLAGS += -MP
 	QMAKE_CXXFLAGS_RELEASE += -Zi
 	QMAKE_CXXFLAGS_RELEASE -= -O2
 	QMAKE_LFLAGS_RELEASE += -debug
 	QMAKE_CLEAN += $(DESTDIR)$(QMAKE_TARGET).pdb
 }
diff --git a/gui/octave-gui.pro b/gui/octave-gui.pro
--- a/gui/octave-gui.pro
+++ b/gui/octave-gui.pro
@@ -1,25 +1,24 @@
 # OctaveGUI - A graphical user interface for Octave
 # Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
 #
 # This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU Affero General Public License as
+# it under the terms of the GNU General Public License as
 # published by the Free Software Foundation, either version 3 of the
 # License, or (at your option) any later version.
 #
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU Affero General Public License for more details.
+# GNU General Public License for more details.
 #
-# You should have received a copy of the GNU Affero General Public License
+# You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-
 # Basic settings:
 QT                  += core gui webkit network	    # Qt modules
 TEMPLATE            = app                           # Build as application
 TARGET              = octave-gui                    # Name of the target binary
 
 DESTDIR             = bin                           # Destination of the output
 UI_DIR              = ui-files                      # Folder for ui files
 MOC_DIR             = moc-files                     # Folder for moc files
diff --git a/gui/src/BrowserWidget.cpp b/gui/src/BrowserWidget.cpp
--- a/gui/src/BrowserWidget.cpp
+++ b/gui/src/BrowserWidget.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "BrowserWidget.h"
 #include <QVBoxLayout>
 #include <QAction>
 #include <QStyle>
 #include <QApplication>
diff --git a/gui/src/BrowserWidget.h b/gui/src/BrowserWidget.h
--- a/gui/src/BrowserWidget.h
+++ b/gui/src/BrowserWidget.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef BROWSERMDISUBWINDOW_H
 #define BROWSERMDISUBWINDOW_H
 
 #include <QWidget>
 #include <QToolBar>
diff --git a/gui/src/CommandLineParser.cpp b/gui/src/CommandLineParser.cpp
--- a/gui/src/CommandLineParser.cpp
+++ b/gui/src/CommandLineParser.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "CommandLineParser.h"
 
 CommandLineParser::CommandLineParser ()
 {
 }
diff --git a/gui/src/CommandLineParser.h b/gui/src/CommandLineParser.h
--- a/gui/src/CommandLineParser.h
+++ b/gui/src/CommandLineParser.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef COMMANDLINEPARSER_H
 #define COMMANDLINEPARSER_H
 
 #include <QList>
 #include <QString>
diff --git a/gui/src/FileEditorMdiSubWindow.cpp b/gui/src/FileEditorMdiSubWindow.cpp
--- a/gui/src/FileEditorMdiSubWindow.cpp
+++ b/gui/src/FileEditorMdiSubWindow.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "FileEditorMdiSubWindow.h"
 #include <QVBoxLayout>
 #include <QApplication>
 #include <QFile>
 #include <QFont>
diff --git a/gui/src/FileEditorMdiSubWindow.h b/gui/src/FileEditorMdiSubWindow.h
--- a/gui/src/FileEditorMdiSubWindow.h
+++ b/gui/src/FileEditorMdiSubWindow.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef FILEEDITORMDISUBWINDOW_H
 #define FILEEDITORMDISUBWINDOW_H
 
 #include "MainWindow.h"
 
diff --git a/gui/src/FilesDockWidget.cpp b/gui/src/FilesDockWidget.cpp
--- a/gui/src/FilesDockWidget.cpp
+++ b/gui/src/FilesDockWidget.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "ResourceManager.h"
 #include "FilesDockWidget.h"
 
 #include <QApplication>
 #include <QFileInfo>
diff --git a/gui/src/FilesDockWidget.h b/gui/src/FilesDockWidget.h
--- a/gui/src/FilesDockWidget.h
+++ b/gui/src/FilesDockWidget.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef FILESDOCKWIDGET_H
 #define FILESDOCKWIDGET_H
 
 #include <QListView>
 #include <QDate>
diff --git a/gui/src/HistoryDockWidget.cpp b/gui/src/HistoryDockWidget.cpp
--- a/gui/src/HistoryDockWidget.cpp
+++ b/gui/src/HistoryDockWidget.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "HistoryDockWidget.h"
 #include <QVBoxLayout>
 
 HistoryDockWidget::HistoryDockWidget (QWidget * parent):QDockWidget (parent)
 {
diff --git a/gui/src/HistoryDockWidget.h b/gui/src/HistoryDockWidget.h
--- a/gui/src/HistoryDockWidget.h
+++ b/gui/src/HistoryDockWidget.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef HISTORYDOCKWIDGET_H
 #define HISTORYDOCKWIDGET_H
 
 #include <QDockWidget>
 #include <QLineEdit>
diff --git a/gui/src/MainWindow.cpp b/gui/src/MainWindow.cpp
--- a/gui/src/MainWindow.cpp
+++ b/gui/src/MainWindow.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include <QMenuBar>
 #include <QMenu>
 #include <QAction>
 #include <QSettings>
 #include <QDesktopServices>
diff --git a/gui/src/MainWindow.h b/gui/src/MainWindow.h
--- a/gui/src/MainWindow.h
+++ b/gui/src/MainWindow.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef MAINWINDOW_H
 #define MAINWINDOW_H
 
 #include <QtGui/QMainWindow>
 #include <QThread>
diff --git a/gui/src/OctaveGUI.cpp b/gui/src/OctaveGUI.cpp
--- a/gui/src/OctaveGUI.cpp
+++ b/gui/src/OctaveGUI.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include <QtGui/QApplication>
 #include <QTranslator>
 #include <QSettings>
 #include "CommandLineParser.h"
 #include "WelcomeWizard.h"
diff --git a/gui/src/ResourceManager.cpp b/gui/src/ResourceManager.cpp
--- a/gui/src/ResourceManager.cpp
+++ b/gui/src/ResourceManager.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "ResourceManager.h"
 #include <QFile>
 #include <QNetworkProxy>
 
 ResourceManager ResourceManager::m_singleton;
diff --git a/gui/src/ResourceManager.h b/gui/src/ResourceManager.h
--- a/gui/src/ResourceManager.h
+++ b/gui/src/ResourceManager.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef RESOURCEMANAGER_H
 #define RESOURCEMANAGER_H
 
 #include <QSettings>
 #include <QDesktopServices>
diff --git a/gui/src/SettingsDialog.cpp b/gui/src/SettingsDialog.cpp
--- a/gui/src/SettingsDialog.cpp
+++ b/gui/src/SettingsDialog.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "ResourceManager.h"
 #include "SettingsDialog.h"
 #include "ui_SettingsDialog.h"
 #include <QSettings>
 
diff --git a/gui/src/SettingsDialog.h b/gui/src/SettingsDialog.h
--- a/gui/src/SettingsDialog.h
+++ b/gui/src/SettingsDialog.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef SETTINGSDIALOG_H
 #define SETTINGSDIALOG_H
 
 #include <QDialog>
 
diff --git a/gui/src/WelcomeWizard.cpp b/gui/src/WelcomeWizard.cpp
--- a/gui/src/WelcomeWizard.cpp
+++ b/gui/src/WelcomeWizard.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "WelcomeWizard.h"
 #include "ui_WelcomeWizard.h"
 
 WelcomeWizard::WelcomeWizard (QWidget *parent) :
   QDialog (parent),
diff --git a/gui/src/WelcomeWizard.h b/gui/src/WelcomeWizard.h
--- a/gui/src/WelcomeWizard.h
+++ b/gui/src/WelcomeWizard.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef WELCOMEWIZARD_H
 #define WELCOMEWIZARD_H
 
 #include <QDialog>
 
diff --git a/gui/src/WorkspaceView.cpp b/gui/src/WorkspaceView.cpp
--- a/gui/src/WorkspaceView.cpp
+++ b/gui/src/WorkspaceView.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "WorkspaceView.h"
 #include <QHBoxLayout>
 #include <QVBoxLayout>
 #include <QPushButton>
 
diff --git a/gui/src/WorkspaceView.h b/gui/src/WorkspaceView.h
--- a/gui/src/WorkspaceView.h
+++ b/gui/src/WorkspaceView.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef WORKSPACEVIEW_H
 #define WORKSPACEVIEW_H
 
 #include <QDockWidget>
 #include <QTreeWidget>
diff --git a/gui/src/backend/OctaveCallbackThread.cpp b/gui/src/backend/OctaveCallbackThread.cpp
--- a/gui/src/backend/OctaveCallbackThread.cpp
+++ b/gui/src/backend/OctaveCallbackThread.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "OctaveCallbackThread.h"
 #include "MainWindow.h"
 
 OctaveCallbackThread::OctaveCallbackThread (QObject * parent)
   : QThread (parent)
diff --git a/gui/src/backend/OctaveCallbackThread.h b/gui/src/backend/OctaveCallbackThread.h
--- a/gui/src/backend/OctaveCallbackThread.h
+++ b/gui/src/backend/OctaveCallbackThread.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef OCTAVECALLBACKTHREAD_H
 #define OCTAVECALLBACKTHREAD_H
 
 #include <QThread>
 #include <QSemaphore>
diff --git a/gui/src/backend/OctaveLink.cpp b/gui/src/backend/OctaveLink.cpp
--- a/gui/src/backend/OctaveLink.cpp
+++ b/gui/src/backend/OctaveLink.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 John P. Swensen, Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "OctaveLink.h"
 
 OctaveLink OctaveLink::m_singleton;
 
 OctaveLink::OctaveLink ():QObject ()
diff --git a/gui/src/backend/OctaveLink.h b/gui/src/backend/OctaveLink.h
--- a/gui/src/backend/OctaveLink.h
+++ b/gui/src/backend/OctaveLink.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 John P. Swensen, Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef OCTAVELINK_H
 #define OCTAVELINK_H
 
 // Octave includes
 #undef PACKAGE_BUGREPORT
diff --git a/gui/src/backend/OctaveMainThread.cpp b/gui/src/backend/OctaveMainThread.cpp
--- a/gui/src/backend/OctaveMainThread.cpp
+++ b/gui/src/backend/OctaveMainThread.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "OctaveMainThread.h"
 #include "OctaveLink.h"
 
 OctaveMainThread::OctaveMainThread (QObject * parent):QThread (parent)
 {
diff --git a/gui/src/backend/OctaveMainThread.h b/gui/src/backend/OctaveMainThread.h
--- a/gui/src/backend/OctaveMainThread.h
+++ b/gui/src/backend/OctaveMainThread.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef OCTAVEMAINTHREAD_H
 #define OCTAVEMAINTHREAD_H
 
 #include <QThread>
 class OctaveMainThread:public QThread
diff --git a/gui/src/backend/ReadlineAdapter.cpp b/gui/src/backend/ReadlineAdapter.cpp
--- a/gui/src/backend/ReadlineAdapter.cpp
+++ b/gui/src/backend/ReadlineAdapter.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "ReadlineAdapter.h"
 
 ReadlineAdapter::ReadlineAdapter (QObject *parent) :
   QObject (parent), command_editor ()
 {
diff --git a/gui/src/backend/ReadlineAdapter.h b/gui/src/backend/ReadlineAdapter.h
--- a/gui/src/backend/ReadlineAdapter.h
+++ b/gui/src/backend/ReadlineAdapter.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef READLINEADAPTER_H
 #define READLINEADAPTER_H
 
 #include "octave/config.h"
 #include "octave/cmd-edit.h"
diff --git a/gui/src/lexer/lexeroctavegui.cpp b/gui/src/lexer/lexeroctavegui.cpp
--- a/gui/src/lexer/lexeroctavegui.cpp
+++ b/gui/src/lexer/lexeroctavegui.cpp
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #include "lexeroctavegui.h"
 #include <qcolor.h>
 #include <qfont.h>
 
 // -----------------------------------------------------
diff --git a/gui/src/lexer/lexeroctavegui.h b/gui/src/lexer/lexeroctavegui.h
--- a/gui/src/lexer/lexeroctavegui.h
+++ b/gui/src/lexer/lexeroctavegui.h
@@ -1,22 +1,22 @@
 /* OctaveGUI - A graphical user interface for Octave
  * Copyright (C) 2011 Jacob Dawid (jacob.dawid@googlemail.com)
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
+ * it under the terms of the GNU General Public License as
  * published by the Free Software Foundation, either version 3 of the
  * License, or (at your option) any later version.
  *
  * This program is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
+ * GNU General Public License for more details.
  *
- * You should have received a copy of the GNU Affero General Public License
+ * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
 #ifndef LEXEROCTAVE_H
 #define LEXEROCTAVE_H
 
 #include "ResourceManager.h"
 #include <QObject>
